spec/system/admin/admin_order_deliver_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Order spec', type: :system do
+  before do
+    driven_by(:rack_test)
+    Bullet.raise = false
+  end
+
+  let!(:employee) { create(:employee) }
+  let!(:admin) { create(:admin_user) }
+  let(:company_value) { create(:company_value) }
+  let!(:reward1) { create(:reward, price: 1) }
+  let!(:kudo) { create(:kudo, giver: employee, receiver: employee, company_value: company_value) }
+  let!(:order) { create(:order, employee_id: employee.id, reward: reward1, reward_snapshot: reward1) }
+
+  it 'tests delivering order' do
+    login_as employee, scope: :employee
```

Comment:
do you really need to login as an employee in this spec?

Reply:
good point, especially that i had in order controller employee authentication that was not needed, but wasn't detected by spec because this login